modules/on-demand-backups-roxctl-admin-pass.adoc

```diff
@@ -34,6 +40,21 @@ $ roxctl -p <admin_password> -e "$ROX_CENTRAL_ADDRESS" central backup
 ----
 $ roxctl -p <admin_password> -e "$ROX_CENTRAL_ADDRESS" central db backup
 ----
+. Run the `backup` command:
```

Comment:
Looks like the `Run the backup command` section is duplicated in this document. Also in the section above in `on-demand-backups-roxctl-admin-pass.adoc` we document that the backup command is different for version 3.0.54 and older, but in this section and in `on-demand-backups-roxctl-api.adoc` as well, we don't mention that. I think we should be consistent, and either don't mention it at all or mention it in both docs.